### DISCUSSION TITLE
Require to sign-in each time with VSCode Remote-SSH

### DISCUSSION THREAD

**Discussion by shraddhabarke** [community/community#9724] [Copilot Conversations]
I have access to Copilot and I'm able to use it successfully on my local VSCode machine. But each time I [remote-SSH](https://code.visualstudio.com/docs/remote/ssh) into a server, Copilot requires me to sign in for authentication. Is there a way to save authentication information in a config file somewhere?

**Comment by eerii** [comment]
The same happens to me, not only with Copilot, but also with synced settings. It happens every time a window is opened for the remote SSH server, and another time when going back to the local window.

**Comment by philkuz** [comment]
Same here. If I miss the toast to reload the window to get it working again.

**Reply by philkuz** [reply]
typo: If I miss the toast *I have to* reload the window to get it working again.

**Reply by saedx1** [reply]
@philkuz  You don't have to, the bell icon on the left bottom corner keeps track of the notifications (unless you pressed 'x' on it).

**Reply by saedx1** [reply]
right bottom*

**Comment by c0de-intention** [comment]
wondering if there is a way to save this to keychain or something for mac
